torch/csrc/jit/serialization/pickler.h

```diff
@@ -7,6 +7,7 @@
 #include <ATen/core/ivalue.h>
 #include <ATen/core/jit_type.h>
 #include <c10/util/ArrayRef.h>
+#include <torch/csrc/WindowsTorchApiMacro.h>
```

Comment:
why do we need a windows header here?

Reply:
This is for `TORCH_API`, which resolves to `CAFFE2_API`, which resolves to `C10_EXPORT`, which resolves to `__attribute__((__visibility__("default")))` on linux, effectively exporting the symbols. Without the `TORCH_API`, `Pickler` and `Unpickler` are not being exported in libtorch.